integration_tests/dbt_project.yml

```
@@ -0,0 +1,70 @@
+config-version: 2
+
+name: 'pendo_source_integration_tests'
+version: '0.1.0'
+
+require-dbt-version: ">=0.20.0"
+profile: 'integration_tests'
+
+vars:
+  pendo_source:
+    account_history: "{{ ref('account_history') }}"
+    application_history: "{{ ref('application_history') }}"
+    event: "{{ ref('event') }}"
+    feature_event: "{{ ref('feature_event') }}"
+    feature_history: "{{ ref('feature_history') }}"
+    group: "{{ ref('group_data') }}"
+    guide_event: "{{ ref('guide_event') }}"
+    guide_history: "{{ ref('guide_history') }}"
+    guide_step_history: "{{ ref('guide_step_history') }}"
+    page_event: "{{ ref('page_event') }}"
+    page_history: "{{ ref('page_history') }}"
+    page_rule_history: "{{ ref('page_rule_history') }}"
+    user: "{{ ref('user') }}"
+    visitor_history: "{{ ref('visitor_history') }}"
+
+    pendo__event_pass_through_columns: []
+    pendo__feature_event_pass_through_columns: []
+    pendo__page_event_pass_through_columns: []
+    pendo__account_history_pass_through_columns: []
+    pendo__visitor_history_pass_through_columns: []
+    
```

For spark to dispatch effectively we will want to add the dispatch configuration to our integration tests.
```suggestion
dispatch:
  - macro_namespace: dbt_utils
    search_order: ['spark_utils', 'dbt_utils']
```